File-Formats/DocIO/Working-with-Find-and-Replace.md

```diff
@@ -2585,4 +2585,6 @@ You can download a complete working sample from [GitHub](https://github.com/Sync
 
 * [How to replace the particular text with hyperlink in Word document](https://www.syncfusion.com/kb/11774/how-to-replace-the-particular-text-with-hyperlink-in-word-document)
 * [How to find the empty paragraphs in the Word document using Essential DocIO](https://www.syncfusion.com/kb/3956/how-to-find-the-empty-paragraphs-in-the-word-document-using-essential-docio)
-* [How to replace text in a Word document with HTML](https://www.syncfusion.com/kb/13627/how-to-replace-text-in-a-word-document-with-html)
\ No newline at end of file
+* [How to replace text in a Word document with HTML](https://www.syncfusion.com/kb/13627/how-to-replace-text-in-a-word-document-with-html)
+* [How to find and replace text inside table in Word document](https://www.syncfusion.com/kb/13640/how-to-find-and-replace-text-inside-table-in-word-document)
```

Comment:
Can you please use this link https://www.syncfusion.com/kb/13640/how-to-find-and-replace-text-inside-table-in-word-document

Reply:
This was the same link used before ,do you mean https://www.syncfusion.com/kb/13641/how-to-find-and-replace-text-inside-table-in-word-document which is for EJ2?